worksheet_A/TerminalHacking.cpp

```diff
@@ -7,6 +7,20 @@
 const int wordLength = 5;
 const int numberOfWords = 15;
 
+int getlikeness(std::string playerguess, std::string secret)
+{
+	int result = 0;
+	for (int check = 0; check < wordLength; check++)
+	{
+		if (playerguess.at(check) == secret.at(check)) {
```

Comment:
USe a consistent formatting style. The rest of the code puts the `{` on the next line, so this should too.